source/apiVolontaria/apiVolontaria/views.py

```diff
@@ -162,7 +162,7 @@ def post(self, request, *args, **kwargs):
                 plain_msg = render_to_string("confirm_sign_up.txt", merge_data)
                 msg_html = render_to_string("confirm_sign_up.html", merge_data)
                 emails_not_sent = services.service_send_mail([request.data["email"]],
-                                                                _("Confirmation d\'enregistrement."),
+                                                                "Confirmation d\'enregistrement.",
```

Comment:
Please use I18N function `_('My subject')` and put an English default sentence. 

Since this line in inside the code, people will not find it and it will be difficult to change.